packages/local-managed-provider/src/local-managed-provider.ts

```diff
@@ -0,0 +1,47 @@
+import { Network, WalletSubProvider } from '@glif/filecoin-wallet-provider'
+import { SignedLotusMessage } from '@glif/filecoin-message'
+import { ExtendedKey, MessageParams } from '@zondax/filecoin-signing-tools'
+import uint8arrays from 'uint8arrays'
+
+const moduleToImport = process.env.JEST_WORKER_ID
+  ? "@zondax/filecoin-signing-tools/nodejs"
+  : "@zondax/filecoin-signing-tools";
+// tslint:disable-next-line:no-var-requires
+const signingTools = require(moduleToImport);
+
+export class LocalManagedProvider implements WalletSubProvider {
+  private readonly privateKey: ExtendedKey
```

Comment:
Does this need to be `private` rather than hiding `privateKey` inside closures via `#privateKey`?

Reply:
Valid concern. Going to redo this with closure.